__docs__/wonder-blocks-tooltip/tooltip.stories.tsx

```diff
@@ -413,6 +396,23 @@ export const AutoUpdate: StoryComponentType = {
                 >
                     Click to update trigger position (fixed)
                 </Button>
+                <Tooltip
```

Comment:
**note:** Moved this instance to the right so it can work more consistently in different environments (browser and JSDOM).

The issue with the previous story was that the assertion in the `play` function was not working properly in headless browsers (e.g. running `npx vitest`) because the viewport/document size was different from the browser one.

This was causing that the Tooltip was not positioned horizontally, causing the test to fail in CI. With this change, we can now ensure that there will be a horizontal reposition.